controllers/localization_controller.go

```diff
@@ -254,8 +254,7 @@ func (r *LocalizationReconciler) reconcile(ctx context.Context, obj *v1alpha1.Lo
 			}
 		}
 		snapshotCR.Spec = v1alpha1.SnapshotSpec{
-			Ref:    strings.TrimPrefix(snapshotName, r.OCIRegistryAddr+"/snapshots/"),
-			Digest: snapshotDigest,
+			Ref: strings.TrimPrefix(snapshotName, r.OCIRegistryAddr+"/"),
```

Comment:
_Something_ has to go in the snapshot creation otherwise we have no idea where to look for things to reconcile. What would that be then? :)

Reply:
Actually I think this is fine. The ref is generated. It's supposed to be the repository name basically. That's not something the snapshot reconciler would care about / find out otherwise.